logsrvd/logsrvd.c

```diff
@@ -1471,6 +1612,9 @@ main(int argc, char *argv[])
 	sudo_fatal(NULL);
     sudo_ev_base_setdef(evbase);
 
+    /* set event timeout from config */
+    timeo.tv_sec = logsrvd_conf_get_sock_timeout();
```

Comment:
As mentioned earlier, I think having logsrvd_conf_get_sock_timeout() return a struct timespec * is a better approach than using a global timeout value.

Reply:
fixed and amended to the corresponding commit